test/README.ssltest.md

```diff
@@ -89,6 +89,9 @@ handshake.
 
 * ExpectedTmpKeyType - the expected algorithm or curve of server temp key
 
+* ExpectedServerKeyType, ExpectedClientKeyType - the expected algorithm or
```

Comment:
You could also just remove ExpectClientCertType if there's no interesting use for it :)